src/cpp/core/CrashHandler.cpp

```diff
@@ -215,11 +217,11 @@ Error initialize(ProgramMode programMode)
       {
          // server mode - default database path to default tmp location
          FilePath tmpPath;
-         Error error = FilePath::tempFilePath(&tmpPath);
+         Error error = FilePath::tempFilePath(tmpPath);
```

Comment:
Is this intentional? We almost never include methods that modify parameters by reference.

Reply:
Yes, the style guidelines for the SDK require that output parameters are provided as references because we can't trust input to the function in the same way that we can in the IDE and it allows us to avoid considerations around improperly initialized pointers. In IDE-only code we still intend to use output pointers and pass to-be-modified values by address.

Reply:
That makes sense. Thanks for clarifying!